lib/blueprinter/helpers/type_helpers.rb

```diff
@@ -8,8 +8,14 @@ def active_record_relation?(object)
         object.is_a?(ActiveRecord::Relation))
     end
 
+    def array_like_class?(object)
+      Blueprinter.configuration.array_like_classes.any? do |klass|
+        object.is_a?(klass)
+      end
+    end
+
     def array_like?(object)
-      object.is_a?(Array) || active_record_relation?(object)
+      object.is_a?(Array) || active_record_relation?(object) || array_like_class?(object)
```

Comment:
would it be possible to natively append `Array` & `ActiveRecord::Relation`(if defined) to the list and use a singular comparison(`array_like_class?`)?

Reply:
great idea! I'll make that update

Reply:
This is done now